dali/kernels/imgproc/resample/params.h

```diff
@@ -62,7 +62,16 @@ struct FilterDesc {
 
 struct ResamplingParams {
   FilterDesc min_filter, mag_filter;
-  int output_size;
+  int output_size = KeepOriginalSize;
+  // just a magic number that's
+  struct AxisROI {
+    AxisROI() = default;
```

Comment:
It's not entirely clear to me, what precisely is a ROI's axis. Is it possible to add such info here?

Reply:
It's a 1D ROI. Will rename to just ROI.